### PR TITLE
Registries

### PR DESCRIPTION
**Pull request by frivoal**
Add process for Registries based on https://github.com/w3c/w3process/issues/168 and TPAC discussions.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/335.html" title="Last updated on Mar 10, 2021, 4:06 PM UTC (ebd4c99)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/335/3ae9f7c...ebd4c99.html" title="Last updated on Mar 10, 2021, 4:06 PM UTC (ebd4c99)">Diff</a>

### CODE REVIEW

**Comment by frivoal** [comment]
Notes:
* This includes (in two separate commits) the variant where the rules and the tables are together in one document, and the one where everything is inside the section of a REC. It does not include the variant where the rules and the tables could live in separate documents: this is more complicated to define, and there wasn't much appetite for it at TPAC.
* An explainer and/or material for `/guide` will also need to be developed, but this is just the process part. Let's take one thing at a time.
* This does not include the rules / guidance on where and how editors drafts should be developed. We need that too, but process-wise, it's a separate edit/issue (see https://github.com/w3c/w3process/issues/322)

**Comment by dwsinger** [comment]
There was a spirited discussion at TPAC that the tables should not be required even to be in /TR, let alone required to be in the same document. Were we even in the same meeting?

**Comment by fantasai** [comment]
There was a lot of confusion about that part, concerns that it would be the same as trying to publish a new FPWD or CR or something like that, or that machine-readable formats wouldn't be allowed, and so there was a lot of pushback at the start of the discussion portion. But once we explained that you can publish the tables in a machine-readable format, and that the entire process can be automated from accepting a GH pull request to posting on /TR using Echidna, people seemed to be on board afaict.

File: index.bs [hunk 1]
```diff
@@ -3909,6 +3930,342 @@ Working Group and Interest Group Notes</h3>
...
+
+		<li>
+			Define the [=rules of the registry=] for each of its [=registry tables=].
+
+		<li>
+			Include the entire contents of each [=registry table=],
+			either inline in the report
+			(e.g. formatted as a table, or list, or other appropriate representation),
+			or in a machine-readable file published as part of the [=technical report=],
+			or (preferably) both.
```
Comment (wseltzer) [inline index.bs]: or as a link to a document or documents published elsewhere.

**Review by Eiygy1992 — APPROVED** [review]

**Pull request merged**